app/controllers/hiring_staff/identifications_controller.rb

```diff
@@ -1,13 +1,29 @@
 class HiringStaff::IdentificationsController < HiringStaff::BaseController
   include ActionView::Helpers::OutputSafetyHelper
 
-  skip_before_action :check_session, only: %i[new create]
-  skip_before_action :check_terms_and_conditions, only: %i[new create]
+  skip_before_action :check_session, only: %i[new create sign_in_by_email]
+  skip_before_action :check_terms_and_conditions, only: %i[new create sign_in_by_email]
   skip_before_action :verify_authenticity_token, only: [:create]
 
   before_action :redirect_signed_in_users
 
-  def new; end
+  def new
+    if EmailSignInFeature.enabled?
+      render :authentication_fallback
```

Comment:
I'd suggest you rename the feature flag to something like this, actually: 

```
if AuthenticationFallback.enabled?
```

It could still live in the FeatureFlag class for now.